### PR TITLE
SQSCANNER-57 - Rename scanner SonarScanner

### PR DESCRIPTION
**Pull request by mickael-caro-sonarsource**

### CODE REVIEW

File: src/main/java/org/sonarsource/scanner/cli/Main.java [hunk 1]
```diff
@@ -71,12 +71,16 @@ void execute() {
       configureLogging(p);
       init(p);
       runner.start();
-      logger.info("SonarQube server " + runner.serverVersion());
+      if(conf.isSonarCloud()) {
+        logger.info("SonarCloud server");
```
[the remark this answers is not in the document]
Comment (mickael-caro-sonarsource) [reply src/main/java/org/sonarsource/scanner/cli/Main.java]: Makes better sense, thanks.

File: src/main/java/org/sonarsource/scanner/cli/Main.java [hunk 2]
```diff
@@ -71,12 +71,16 @@ void execute() {
       configureLogging(p);
       init(p);
       runner.start();
-      logger.info("SonarQube server " + runner.serverVersion());
+      if(conf.isSonarCloud()) {
```
[the remark this answers is not in the document]
Comment (mickael-caro-sonarsource) [reply src/main/java/org/sonarsource/scanner/cli/Main.java]: About the space between the if and openning parenthesis ? I did a refactor, this is the only thing fixed.

File: src/main/java/org/sonarsource/scanner/cli/Conf.java [hunk 3]
```diff
@@ -68,6 +70,11 @@ Properties properties() {
     return result;
   }
 
+  boolean isSonarCloud(){
```
Comment (henryju) [inline src/main/java/org/sonarsource/scanner/cli/Conf.java]: Missing space here before curly brace

File: src/main/java/org/sonarsource/scanner/cli/Conf.java [hunk 4]
```diff
@@ -71,7 +72,7 @@ Properties properties() {
 
   boolean isSonarCloud(){
     String hostUrl = properties().getProperty(PROPERTY_SONAR_HOST_URL);
-    return hostUrl.toLowerCase().contains("sonarcloud") ? true : false;
+    return hostUrl.toLowerCase(Locale.getDefault()).contains("sonarcloud") ? true : false;
```
Comment (henryju) [inline src/main/java/org/sonarsource/scanner/cli/Conf.java]: Is `hostUrl` garanteed to be non null at this point?

Comment (mickael-caro-sonarsource) [reply src/main/java/org/sonarsource/scanner/cli/Conf.java]: I suppose that this is not guaranteed, but functionnaly speaking, as long as we are already on the CLi, it should be filled (is there any mechanism that fills this property ? ). But, we never know, so i should defo add a nullcheck here for sure.

**Comment by mickael-caro-sonarsource** [comment]
@henryju i'm struggling with the coverage on Next (issue with the bug has been fixed, was the position of the block not matching the one on master, due to auto-refactor of the code), i'm pretty sure that my new method is fully covered (coverage is ok on Intellij and local jacoco), but still stuck at 84.6 here.

**Pull request merged**